vector/src/main/java/im/vector/app/features/location/live/map/LiveLocationBottomSheetController.kt

```diff
@@ -26,8 +26,6 @@ import im.vector.app.core.resources.StringProvider
 import im.vector.app.core.resources.toTimestamp
 import im.vector.app.core.time.Clock
 import im.vector.app.features.home.AvatarRenderer
-import im.vector.app.features.location.live.map.bottomsheet.LiveLocationUserItem
-import im.vector.app.features.location.live.map.bottomsheet.liveLocationUserItem
```

Comment:
Was it unused import? Wondering why ktlint did not complain about it.

Reply:
(The code does not compile on my side if I just remove these 2 lines on my current branch, but the CI seems to be happy here. Please double check!)

Reply:
[LiveLocationUserItem](https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/features/location/live/map/LiveLocationUserItem.kt#L17) has the wrong package declared for the file location

Reply:
https://github.com/vector-im/element-android/pull/6415#discussion_r910737101